hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/web/ozShell/bucket/AddAclBucketHandler.java

```diff
@@ -88,10 +88,15 @@ public Void call() throws Exception {
             OzoneObj.StoreType.valueOf(storeType))
         .build();
 
-    boolean result = client.getObjectStore().addAcl(obj,
+    boolean aclExisted = client.getObjectStore().checkAclExist(obj,
```

Comment:
I think we don't need a checkAclExist

addAcl semantic true, acl is added successfully, false means acl already exist that is why we cannot add it. So, no need of checkAclExist()